cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -382,8 +411,8 @@ func (c *Client) AddMember(name string, peerAddrs string) ([]Member, error) {
 	}
 
 	// Returns the updated list of etcd members
-	ret := []Member{}
-	for _, m := range resp.Members {
+	var ret []Member
```

Comment:
should we preserve the original `ret := []Member{}` to avoid returning a `nil` slice?

Reply:
I didn't think this would be necessary, but to preserve contract, I reverted this change.